pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = -p no:warnings
```

Comment:
Would rather warnings show so that we can fix them as needed

Reply:
removed